unity-renderer/Assets/DCLServices/WearablesCatalogService/LambdasWearablesCatalogService.cs

```diff
@@ -397,52 +399,75 @@ private async UniTask<WearableItem> SyncWearablesRequestsAsync(string newWearabl
             return result.FirstOrDefault(x => x.id == newWearableId);
         }
 
-        private static IList<WearableDefinition> ValidateWearables(IList<WearableDefinition> wearablesFromLambdas)
+        private List<WearableItem> ValidateWearables(IEnumerable<WearableWithEntityResponseDto.ElementDto> wearableElements)
         {
-            var invalidWearablesIndices = ListPool<int>.Get();
+            List<WearableItem> wearables = new ();
 
-            for (var i = 0; i < wearablesFromLambdas.Count; i++)
+            foreach (var item in wearableElements)
             {
-                var item = wearablesFromLambdas[i];
-                var definition = item.definition;
+                WearableWithEntityResponseDto.ElementDto.EntityDto entity = item.entity;
+                WearableWithEntityResponseDto.ElementDto.EntityDto.MetadataDto metadata = entity.metadata;
 
-                if (FilterInvalidWearableItem(item, i, invalidWearablesIndices))
+                if (IsInvalidWearable(metadata))
                     continue;
 
                 try
                 {
-                    foreach (var representation in definition.data.representations)
+                    WearableItem wearable = new WearableItem
                     {
-                        foreach (var representationContent in representation.contents)
-                            representationContent.hash = representationContent.url[(representationContent.url.LastIndexOf('/') + 1)..];
+                        data = new WearableItem.Data
+                        {
+                            representations = new WearableItem.Representation[metadata.data.representations.Length],
+                            category = metadata.data.category,
+                            hides = metadata.data.hides,
+                            replaces = metadata.data.replaces,
+                            tags = metadata.data.tags,
+                        },
+                        baseUrl = TEXTURES_URL_ORG,
```

Comment:
should i use `$"{catalyst.contentUrl}/contents/"` instead?

Reply:
I've fixed it. We were using a hardcoded node to fetch wearables content, which was really bad. I've replaced it with the content server of the catalyst